ttyplot.c

```diff
@@ -172,6 +175,14 @@ int main(int argc, char *argv[]) {
             case 'm':
                 hardmax=atof(optarg);
                 break;
+            case 'M':
+                hardmin=atof(optarg);
+                int i;
+                for(i=0;i<1024;i++){
+                    values1[i]=hardmin;
+                    values2[i]=hardmin;
+                }
+                break;
```

Comment:
I think also that if hardmin is set also softmax should be set to hardmin+1 or so. If you run `ttyplot -M 10` and enter few values < 10 in the beginning the plot will behave weirdly because min=max. If you run `ttyplot -M 10 -s 11` it will work much better.

Reply:
Added the check after opt parsing.